model/trace/llm.yaml

```diff
@@ -0,0 +1,176 @@
+    attributes:
+      - ref: llm.completion
+        requirement_level: recommended
+        note: >
+          The full response string from an LLM. If the LLM responds with a more complex output like a JSON object made up of several pieces (such as OpenAI's message choices), this field is the content of the response. If the LLM produces multiple responses, then this field is left blank, and each response is instead captured in an event determined by the specific LLM technology semantic convention.
+
+  - id: llm.openai
+    type: span
+    brief: >
+      These are the attributes when instrumenting OpenAI LLM requests with the `/chat/completions` endpoint.
+    attributes:
+      - ref: llm.request.vendor
+        requirement_level: recommended
+        examples: ['openai', 'microsoft']
+        tag: tech-specific-openai-request
+      - ref: llm.request.model
+        requirement_level: required
+        note: >
+            The name of the LLM a request is being made to. If the LLM is supplied by a vendor, then the value must be the exact name of the model requested. If the LLM is a fine-tuned custom model, the value should have a more specific name than the base model that's been fine-tuned.
+        tag: tech-specific-openai-request
+      - ref: llm.request.max_tokens
+        tag: tech-specific-openai-request
+      - ref: llm.request.temperature
+        tag: tech-specific-openai-request
+      - ref: llm.request.top_p
+        tag: tech-specific-openai-request
+      - ref: llm.request.stream
+        tag: tech-specific-openai-request
+      - ref: llm.request.stop_sequences
+        tag: tech-specific-openai-request
+      - ref: llm.request.openai.presence_penalty
+        tag: tech-specific-openai-request
+      - ref: llm.request.openai.logit_bias
+        tag: tech-specific-openai-request
+      - ref: llm.request.openai.user
+        tag: tech-specific-openai-request
+      - ref: llm.request.openai.response_format
+        tag: tech-specific-openai-request
+      - ref: llm.request.openai.seed
+        tag: tech-specific-openai-response
+      - ref: llm.response.id
+        tag: tech-specific-openai-response
+      - ref: llm.response.finish_reason
+        tag: tech-specific-openai-response
+      - ref: llm.usage.prompt_tokens
+        tag: tech-specific-openai-response
+      - ref: llm.usage.completion_tokens
+        tag: tech-specific-openai-response
+      - ref: llm.usage.total_tokens
+        tag: tech-specific-openai-response
+      - ref: llm.response.openai.created
+        tag: tech-specific-openai-response
+      - ref: llm.response.openai.system_fingerprint
+        tag: tech-sepecifc-openai-response
+    events:
+      - llm.content.openai.prompt
+      - llm.content.openai.tool
+      - llm.content.openai.completion.choice
+
+  - id: llm.content.openai.prompt
```

Comment:
do we need this event?
It can be `llm.content.prompt` with open-ai specific attributes

Reply:
That's a good question. Given that OpenAI (specifically) has a really different way of modeling prompts and completions, I wonder if it won't be cumbersome to use the same event for both?

Reply:
we can start with one event and add more once it's proven to be too difficult. the spec would stay experimental for now anyway. 
For the time being we can just list OpenAI-specific attributes and mention that they would appear on the events.

(Unless you already have good reasons to keep events separate)

Reply:
Some frameworks (e.g. [vllm](https://docs.vllm.ai/en/latest/getting_started/quickstart.html#openai-compatible-server)) have OpenAI-compatible serving APIs.

Using the same event name can benefit this use case.